apps/citizen-func/GetMessage/handler.ts

```diff
@@ -94,7 +94,7 @@ type IGetMessageHandler = (
 const getErrorOrPaymentData = async (
   context: Context,
   serviceModel: ServiceModel,
-  redisClient: redis.RedisClientType,
+  redisClient: TE.TaskEither<Error, redis.RedisClientType>,
```

Comment:
rename to `redisClientTask`?

Reply:
[03f5440](https://github.com/pagopa/io-messages/pull/12/commits/03f5440d98abe5c1e384f39b0020591e832276c3)